articles/fleet-4.67.0.md

```diff
@@ -0,0 +1,40 @@
+# Fleet 4.67.0 | Foreign vitals, policy targets, cancel activities
+
+<div purpose="embedded-content">
+   <iframe src="TODO" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Fleet 4.67.0 is now available. See the complete [changelog](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.67.0) or read on for highlights. For upgrade instructions, visit the [upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs.
+
+## Highlights
+
+- Foreign vitals
+- Policy targets
+- Cancel activities
+
+### Foreign vitals
+
+Fleet now pulls end user details from your identity provider (IdP)—like IdP email, full name, and group memberships—into host vitals. This makes it easier to identify who is using each host to speed up troubleshooting and audits. Learn more [here](https://fleetdm.com/guides/foreign-vitals-map-idp-users-to-hosts).
+
+### Policy targets
+
+Security engineers can now target policies to specific hosts using labels. This gives teams more precise control over enforcement within a [Fleet team](https://fleetdm.com/guides/teams), helping apply the right checks to the right hosts.
```

Comment:
```suggestion
Security engineers can now target policies to specific hosts using labels. This gives teams more precise control over enforcement within a [Fleet team](https://fleetdm.com/guides/teams), helping apply the right checks to the right hosts.

Heads up, if you're using [GitOps](https://fleetdm.com/docs/configuration/yaml-files), policy targets via labels isn't for production quite yet because of [this bug](https://github.com/fleetdm/fleet/issues/28154). Fix is coming in Fleet 4.67.1.
```